Yosemite/Yosemite/Stores/OrderStore.swift

```diff
@@ -281,6 +281,7 @@ private extension OrderStore {
             handleOrderItems(readOnlyOrder, storageOrder, storage)
             handleOrderCoupons(readOnlyOrder, storageOrder, storage)
             handleOrderShippingLines(readOnlyOrder, storageOrder, storage)
+            handleOrderRefundsCondensed(readOnlyOrder, storageOrder, storage)
```

Comment:
suggestion: could add a unit test to verify this change

Reply:
Will do!